tests/accelerators/test_accelerator_connector.py

```diff
@@ -506,6 +506,26 @@ def test_accelerator_cpu(_):
         trainer = Trainer(accelerator="cpu", gpus=1)
 
 
+@mock.patch("torch.cuda.is_available", return_value=True)
+@mock.patch("torch.cuda.device_count", return_value=2)
+@pytest.mark.parametrize("gpu_count", (["0"], [0, "1"], ["GPU"], [["0", "1"], [0, 1]], [False], False))
+def test_accelererator_invalid_type_gpus(mock_is_available, mock_device_count, gpu_count):
```

Comment:
mock order is inverted. the patch decorator closes to the bottom corresponds to the first input argument. for parametrizations, it doesn't matter due to their inspection of named variables.

Reply:
Ah, good catch! Thanks, Adrian. Fixed now.